bamboo/bootstrap-publish.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+. ./bamboo/set-bamboo-env-variables.sh
+if [[ $PUBLISH_FLAG == true ]]; then
```

Comment:
PUBLISH_FLAG is pulled from the bamboo job values.     To run deployment, we'll create a deployment branch plan that sets this value to true.